src/main/java/frc/robot/subsystems/lights/Lights.java

```diff
@@ -107,5 +107,32 @@ public void idleAnimation(int length) {
             });
         }
     }
+
+    public void idleAutonDriveAnimation(int length) {
```

Comment:
Instead of creating a whole new method with the same code, can you just make the idle animation code accept two colors? We don't want to duplicate code.

Reply:
e.g. `idleAnimation(int length, Color8Bit stripColor, Color8Bit animColor)` or something

Reply:
did that, and created get color methods in lights to allow lights controller to use colors. Could change to one method using enums, but I'm not going to do that unless y'all really want me to.